src/Symfony/Component/ErrorRenderer/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -11,33 +11,34 @@
 
 namespace Symfony\Component\ErrorRenderer\ErrorRenderer;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Component\ErrorRenderer\Exception\FlattenException;
-use Symfony\Component\HttpKernel\Debug\FileLinkFormatter;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Log\DebugLoggerInterface;
 
 /**
  * @author Yonel Ceruto <yonelceruto@gmail.com>
  */
 class HtmlErrorRenderer implements ErrorRendererInterface
 {
-    private const GHOST_ADDONS = [
-        '02-14' => self::GHOST_HEART,
-        '02-29' => self::GHOST_PLUS,
-        '10-18' => self::GHOST_GIFT,
-    ];
```

Comment:
This is a removed "feature", right? As the PR is huge, can you tell us which other "features" have been removed?

Reply:
Opps! sorry, that wasn't intentional. (re-added).

I've moved all the features from `exception_full.html.twig` to here, in practice, this has more features than the current one, as it was more advanced.